.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = library/securimage
 	url = https://github.com/mantisbt/securimage.git
 	branch = mantis
+[submodule "library/parsedown"]
+	path = library/parsedown
+	url = https://github.com/erusev/parsedown.git
+	branch = mantisbt
```

Comment:
Thanks you @dregad, I assumed we can directly include the submodule from its original location.  

Let me know if you already fork the repo to mantisbt.org.

Thanks again!